libcontainerd/client_windows.go

```diff
@@ -614,13 +630,8 @@ func (clnt *client) Pause(containerID string) error {
 		return err
 	}
 
-	for _, option := range container.options {
-		if h, ok := option.(*HyperVIsolationOption); ok {
-			if !h.IsHyperV {
-				return errors.New("cannot pause Windows Server Containers")
-			}
-			break
-		}
+	if container.ociSpec.Windows.HyperV == nil {
```

Comment:
Actually, I plead temporary insanity. Ignore that comment. It's fine as is! I misread `==` as `!=`.